### PR TITLE
feat(OCPADVISOR-128): workloads table sorting

### PR DESCRIPTION
**Pull request by Fewwy**
How to check the PR

Make sure that the columns are sortable

### CODE REVIEW

File: src/Components/WorkloadsListTable/WorkloadsListTable.js [hunk 1]
```diff
@@ -80,12 +85,49 @@ const WorkloadsListTable = ({
...
+          case WORKLOADS_TABLE_SEVERITY:
+            fst = Object.values(a.metadata.hits_by_severity || 0).reduce(
+              (a, b) => a + b,
+              0
+            );
+            snd = Object.values(b.metadata.hits_by_severity || 0).reduce(
+              (a, b) => a + b,
+              0
+            );
+            return fst > snd ? d : snd > fst ? -d : 0;
```
Comment (AsToNlele) [inline src/Components/WorkloadsListTable/WorkloadsListTable.js:118]: You could do something like this

```suggestion
          case WORKLOADS_TABLE_SEVERITY:
            fst = a.metadata.hits_by_severity || { 1: 0, 2: 0, 3: 0, 4: 0 };
            snd = b.metadata.hits_by_severity || { 1: 0, 2: 0, 3: 0, 4: 0 };

            for (let i = 4; i > 0; i--) {
              if (fst[i] > snd[i]) return d;
              if (fst[i] < snd[i]) return -d;
            }

            return 0;
```

**Review by AsToNlele — CHANGES_REQUESTED** [review]
Just to verify, was it confirmed the sorting for severity should be the same like in vulnerability? If that's the case I left a comment

File: src/Components/WorkloadsListTable/WorkloadsListTable.js (hunk 1 above)
Comment (Fewwy) [reply src/Components/WorkloadsListTable/WorkloadsListTable.js:118]: Ok, gonna do that :)

File: src/Components/WorkloadsListTable/WorkloadsListTable.js [hunk 2]
```diff
@@ -98,8 +140,16 @@ const WorkloadsListTable = ({
...
-              <p key={`${index}-namespace`}>{item.namespace.name}</p>
+              {item.cluster.display_name ? (
+                <p key={`${index}-cluster`}>{item.cluster.display_name}</p>
+              ) : (
+                <p key={`${index}-cluster`}>{item.cluster.uuid}</p>
+              )}
+              {item.namespace.name ? (
+                <p key={`${index}-cluster`}>{item.namespace.name}</p>
+              ) : (
+                <p key={`${index}-cluster`}>{item.namespace.uuid}</p>
```
Comment (AsToNlele) [inline src/Components/WorkloadsListTable/WorkloadsListTable.js:151]: Typo here, so we can get rid of the same key errors

```suggestion
                <p key={`${index}-namespace`}>{item.namespace.name}</p>
              ) : (
                <p key={`${index}-namespace`}>{item.namespace.uuid}</p>
```

**Comment by Fewwy** [comment]
> Just to verify, was it confirmed the sorting for severity should be the same like in vulnerability? If that's the case I left a comment

Yeah, pretty much that's the behavior we can copy

**Review by AsToNlele — APPROVED** [review]
LGTM @Fewwy, thanks for the quick fix :smile:

**Pull request merged**